File: skill-packs/java/quarkus/quarkus-cloud-native-apis/examples/good-example.java
// GOOD — JAX-RS resource with CDI injection and DTO response
@Path("/api/v1/orders")
@ApplicationScoped
public class OrderResource {
    @Inject OrderService orderService;

    @POST
    @Consumes(APPLICATION_JSON)
    @Produces(APPLICATION_JSON)
    public Response create(@Valid CreateOrderRequest request) {
        OrderResponse body = orderService.create(request);
        return Response.status(CREATED).entity(body).build();
    }
}

File: skill-packs/python/fastapi/fastapi-async-backend/examples/good-example.py
# GOOD — async route, Pydantic models, dependency injection
@router.post("/orders", response_model=OrderResponse, status_code=201)
async def create_order(
    body: CreateOrderRequest,
    service: OrderService = Depends(get_order_service),
) -> OrderResponse:
    return await service.create(body)
